src/udev/udev-rules.c

```diff
@@ -1321,6 +1321,16 @@ int udev_rules_parse_file(UdevRules *rules, const char *filename) {
         return 0;
 }
 
+unsigned int udev_check_current_rule_file(UdevRules *rules) {
```

Comment:
Please drop `int`, that is `unsigned int` -> `unsigned`.

Reply:
Ouch, that's unusual.  I haven't seen anything about it in the docs.
Is this a policy?  If yes, I suppose I can document it somewhere in `docs/CODING_STYLE.md`.

Reply:
Posted a patch at #26718.

Reply:
Thank you!